### ISSUE TITLE
Add New Dialect: Russian (Python)

### ISSUE THREAD

**Issue by madiedgar** [legesher/legesher-translations#20]
## Adding Russian to the Legesher Translations for Python
Version 1 of Legesher only includes translations for the Python programming language. In order to add this language to the project, complete the steps detailed in [the translation guide](https://legesher.readme.io/docs/translation-guide)

**To complete this issue**:
- [x] Create Russian dialect `.yml` file
- [x] Enter translation for _at least_ the Python keywords
- [x] Merge changes into branch titled `russian-translation` on this repo

### Reviewers Needed
In order to add the translations into our database to use, the translations will need to be approved and accepted by 3 additional native / fluent speakers. If you would like to be one of the reviewers for this language, please mention in this issue! :smile:

**Comment by OleksiyRudenko** [comment]
@madiedgar can you please clarify the translation strategy?
English version just dubs the keywords. From the other hand literal translation of terms like `elif` doesn't make much sense 'cause `elif` is not a word from the natural English language. Should the translation be an explanation of the keyword?

Which of the options below would match better?
```
  "elif": "is an abbreviation of 'else if'"
  "elif": "is short for 'else if' and denotes a condition to check when parent 'if' statement is False"
```

It seems like this unclarity might be a reason for not many contributors willing to take the issue.

P.S. I might be a reviewer for Russian and Ukrainian.

**Comment by madiedgar** [comment]
@OleksiyRudenko would love to know your thoughts for words like `elif`! Want to help us get started for Russian? 👍 

Also Ukrainian is almost finished with it's review phase, would love your help to complete it! 😄

**Comment by madiedgar** [comment]
@startonmonday oops, mentioned you on the `Review Dialect` issue for Russian (#38) when this one is still open. This would be where you could start! 👍

**Comment by KChantal** [comment]
Hi @madiedgar ! I noticed that this issue has been a little quiet. I'm not entirely native in Russian, but I could certainly help to get the translation started, if it hasn't been claimed or started yet?

**Comment by madiedgar** [comment]
Thanks @KChantal! That would be amazing 👏Go for it! 😄

**Comment by madiedgar** [comment]
Closing this issue as it was resolved by #84 by @KChantal 🎉
